rootfs-builder/euleros/config.sh

```diff
@@ -22,3 +22,5 @@ INIT_PROCESS=systemd
 # List of zero or more architectures to exclude from build,
 # as reported by  `uname -m`
 ARCH_EXCLUDE_LIST=()
+# Allow the build to fail without generating an error
```

Comment:
Could you add a ref to a URL explaining why we allow this distro to fail?

Reply:
Thanks @jodh-intel for the review.